src/Explore.Cli/Program.cs

```diff
@@ -27,6 +27,15 @@ public static async Task<int> Main(string[] args)
         var filePath = new Option<string>(name: "--file-path", description: "The path to the file used for importing data") { IsRequired = true };
         filePath.AddAlias("-fp");
 
+        var exportPath = new Option<string>(name: "--export-path", description: "The path for exporting files. It can be either a relative or absolute path") { IsRequired = false };
+        exportPath.AddAlias("-ep");
+
+        var exportFileName = new Option<string>(name: "--export-name", description: "The name of the file to export") { IsRequired = false };

```

Comment:
Perhaps we could just use `file-path` also for this purpose and just enhance the description such that it covers both?

Reply:
I originally considered using file-path. However, file-path is marked as required. Did you want the export path to be required or did you want me to change it so that file-path is not required? In my mind, it didn't make sense to have file-path not required as it is used for importing. But also, it didn't make sense to force the user to always provide an export path

Reply:
I'd just create a new option variable for the export use case, and have the Option.name = file-path, and isRequired = false.

Example:
```
var exportFilePath = new Option<string>(name: "--file-path", description: "The path to the file used for importing data") { IsRequired = false };
filePath.AddAlias("-fp");
```